### ISSUE TITLE
Query has no results to return

### ISSUE THREAD

**Issue by EHulburd** [ecirpnaes/SimpleDataScripter#6]
Whenever I try to use this, I just get an error of "Query has no results to return". I've tried with several different tables, and the tables definitely do have data in, but i get the same error each time

**Comment by MartijnSchoemaker** [comment]
Since a few weeks I have the same issue. It worked before...

**Comment by ecirpnaes** [comment]
What version of the DataScripter extension are you using? (The latest is
v0.1.3).
What version of Azure Data studio are you using?

**Comment by MartijnSchoemaker** [comment]
![Schermafbeelding 2020-05-13 om 15 33 49](https://user-images.githubusercontent.com/2683569/81819110-309fc500-952f-11ea-81b0-771b4e19c848.png)
![Schermafbeelding 2020-05-13 om 15 33 28](https://user-images.githubusercontent.com/2683569/81819114-31385b80-952f-11ea-83b5-5cfd1817c6c7.png)

**Comment by ecirpnaes** [comment]
Can you ensure the actual version you have installed in the latest version
by checking under the [Enabled] tab in your extensions list? (See
attachment).
If it is, can you then test the sql you are trying to script by copying it
out of the input window and running it directly in an Azure SQL
editor window?

**Comment by MartijnSchoemaker** [comment]
Yes it is version 0.1.3. I already uninstalled, downloaded vsix and installed again several times.
The sql I'm trying to run is the default 'SELECT' statement it shows. Copying this and run it manually in editor is working.

**Comment by EHulburd** [comment]
Yes I was also using 0.1.3. And the select statement worked fine in an editor.

**Comment by ecirpnaes** [comment]
This is a hard one for me to troubleshoot. I am passing the query directly
to the Azure queryProvider and catching any return errors. Unfortunately,
the error I display to the user is exactly what is returned from the api
call. There is no additional information available. It would be nice if
there were something useful like "invalid credentials", "invalid sql",
etc..., but MS has not seen fit to provide that in the error object.
I'll see if I can add additional debugging options but it's not going to be
a quick fix.

**Comment by LukeStephen** [comment]
I am seeing this too, worked yesterday when I installed it, now I get no results anytime I try to run it, no matter what the SQL statement is

**Comment by dtotzke** [comment]
Doesn't work when connected to an Azure SQL Database using "Azure Active Directory - Universal with MFA support" as the Authentication type.  I switched the connection to use a SQL login and it works fine.

**Comment by gte445e** [comment]
Choosing the Database, instead of leaving `<Default>`, in the Connection Details worked for me
